Migration script/openIMIS migration v1.4.2 - v1.4.3.sql

```diff
@@ -0,0 +1,1097 @@
+--- MIGRATION Script from v1.4.2 to v1.4.3
```

Comment:
In OTC-111 you named the migration file openIMIS migration v1.4.2 - tbd.sql. Please use same file here.

Reply:
I moved to a new file called `openIMIS migration v1.4.2 - tbd.sql`